lib/addons/p5.dom.js

```diff
@@ -736,9 +737,11 @@
    *
    * </code></div>
    */
-  p5.prototype.createInput = function(value) {
+  p5.prototype.createInput = function(value, type) {
     var elt = document.createElement('input');
-    elt.type = 'text';
+    if (type) elt.type = type;
```

Comment:
Sorry, Didnt realize I could edit the pull request. I implemented your code change. This is all very new to me.